### ISSUE TITLE
7.24.0+ Breaks everything

### ISSUE THREAD

**Issue by boris-petrov** [babel/ember-cli-babel#389]
Upgrading to 7.24.0 or 7.25.0 leads to a bunch of errors like:

```
Error: Could not find module `lodash/identity` imported from `...`
```

Everything else is the latest possible version. Do you need a reproduction?

**Comment by makepanic** [comment]
Do you use ember-cli-typescript? It might be related to https://github.com/ef4/ember-auto-import/issues/359

**Comment by boris-petrov** [comment]
I do use it, yes. It's possible, I guess. Thanks, I'll subscribe to the other issue too!

**Comment by wagenet** [comment]
Also seeing this.

**Comment by rwjblue** [comment]
Fix released in 7.26.1

**Comment by wagenet** [comment]
Thanks!
